### PR TITLE
Refactor bulk app translations

### PR DESCRIPTION
**Pull request by dannyroberts**
to give it an organization less likely to have a headers-don't-match-rows problem like the one that happened recently. Also added another test.

### CODE REVIEW

File: corehq/apps/app_manager/tests/test_bulk_app_translation.py [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import codecs
+from collections import OrderedDict
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_bulk_app_translation.py]: F401 'OrderedDict' imported but unused

**Comment by czue** [comment]
build failed:
```
======================================================================
ERROR: setUpClass (corehq.apps.app_manager.tests.test_bulk_app_translation.BulkAppTranslationDownloadTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_bulk_app_translation.py", line 143, in setUpClass
    for ws in wb_reader.worksheets]
AttributeError: 'WorksheetJSONReader' object has no attribute 'title'
----------------------------------------------------------------------
```

**Comment by dannyroberts** [comment]
Oops, depends on https://github.com/dimagi/dimagi-utils/pull/252

**Comment by dannyroberts** [comment]
@NoahCarnahan

**Comment by NoahCarnahan** [comment]
Looks good, thanks Danny.

**Pull request merged**